src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -386,4 +386,7 @@ The first part date {1} should be lesser than or equal to the second part date {
     <value>The extended query tag '{0}' is expected to have VR '{1}' but has '{2}' in file.</value>
     <comment>{0}  dicom tag. {1} expected vr code. {2} vr in file.</comment>
   </data>
+  <data name="ExtendedQueryTagVersionMismatch" xml:space="preserve">
+    <value>Version of extended query tag mismatch.</value>
```

Comment:
nit: maybe like `One or more extended query tags have been added.`?

Reply:
I changed to `One or more extended query tags have been modified.`